crates/solver/src/in_flight_orders.rs

```diff
@@ -89,6 +93,19 @@ impl InFlightOrders {
                 u256_to_big_uint(&order.data.buy_amount) > order.metadata.executed_buy_amount
             }
         });
+
+        tracing::debug!(
+            auction_block = %auction.block,
+            latest_settlement_block = %auction.latest_settlement_block,
+            inflight_before = %inflight_before.len(),
+            inflight_before = ?inflight_before,
```

Comment:
nit: Listing the order UIDs might get long. I would suggest to move all the short values (blocks and lengths) to the front so the logs shows as much information as possible on a first glance without having to expand the log in kibana.